client/client_test.go

```diff
@@ -1285,6 +1376,11 @@ func TestOpenFgaClient(t *testing.T) {
 		if len(*got.Assertions) != len(*expectedResponse.Assertions) || (*got.Assertions)[0].Expectation != (*expectedResponse.Assertions)[0].Expectation {
 			t.Fatalf("OpenFgaClient.%v() = %v, want %v", test.Name, string(responseJson), test.JsonResponse)
 		}
+		// ReadAssertions without options should work
+		_, err = fgaClient.ReadAssertions(context.Background()).Execute()
+		if err != nil && err.Error() != "Required parameter AuthorizationModelId was not provided" {
```

Comment:
do we ever expect `Required parameter AuthorizationModelId was not provided`?

Reply:
Yep, in this case because the model id was not provided

Reply:
Should we fail if err is nil then?